### ISSUE TITLE
Load on re-focus

### ISSUE THREAD

**Issue by anthonyvialleton** [mbenford/ngTagsInput#251]
We would appreciate to have a way to *avoid* *focusing + unfocusing* to *reload* the tags list.

Here is the simple example that need this action to regenerate tags list typeahead

```
<tags-input class="input-group" placeholder="Search by Tags" add-from-autocomplete-only="true" ng-model="searchform.tags" on-tag-added="search()" on-tag-removed="search()" display-property="name" min-length="1" >
                        <auto-complete source="loadPromptTags($query)" load-on-focus="true" load-on-empty="true" min-length="1"></auto-complete>
 </tags-input>
```

May be there is a simple way to do it from version 2.1.1 ?

**Comment by mbenford** [comment]
This has been fixed in v2.2.0 by making the `loadOnEmpty` option behave correctly. No new options needed. :) 

I believe this issue can be closed now, right?

**Comment by alsoscotland** [comment]
Yes thanks
